content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,276 @@ keywords:
+{{< include file="/static/includes/apps/AppsWizardAppNameAndVersion.md" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/LocateNextcloudAppWidget.png" alt="Locate Nextcloud App Widget" id="Locate Nextcloud App Widget" >}}
+Next, enter the **Nextcloud Configuration** settings.
+{{< expand "Entering Nextcloud Configuration Settings" "v" >}}
+For a basic installation, you can accept default values, but enter the following values:
+Enter the name of the administration user in **Admin User** and **Admin Password**.
+If using an existing Nextcloud account, enter the administration credentials for that account or enter new to create new Nextcloud user account credentials.
 
-   Click on the widget to open the **Nextcloud** details screen, then click **Install**.
-   If this is the first application installed, TrueNAS displays a dialog about configuring apps.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfig1.png" alt="Nextcloud Configuration Username" id="Nextcloud Configuration Username" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/AppsInformationDialog.png" alt="Apps Information Dialog" id="Apps Information Dialog" >}}
+Select the APT packages you want to use. Nextcloud requires ffmpeg and smbclient. Click **Add** to the right of **APT Packages** twice to add two sets of **Package** fields.
+Select **ffmpeg** in one, and **smbclient** in the other.
+If also selecting **ocrmypdf**, also select the **Tesseract Language Codes** option you want to use. Click **Add** to show the **Language** field then select either **chi-sim** for simplified Chinese or **eng** for English.
 
-   Click **Confirm** then **Agree** to close the dialog and open the **Nextcloud** details screen opens.
+Enter the IP address and port for your TrueNAS system in **Host** as* ##.###.###.##:port#*. The **Data Directory Path** is pre-populated with the correct path.
 
-   {{< trueimage src="/images/SCALE/Apps/NextcloudAppDetailsScreen.png" alt="Nextcloud Details Screen" id="Nextcloud Details Screen" >}}
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfig2.png" alt="Enter Host and Other Config Settings" id="Enter Host and Other Config Settings" >}}
 
-   Click **Install** to open the **Install Nextcloud** wizard.
+Enter a password in **Redis Password** to create a new credential or enter the existing password if you already have Redis configured in your Nextcloud account.
+Enter a password in **Database Password** to create a new credential for the Nextcloud database or enter the existing password if you already have the Nextcloud account database configured.
 
-   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudScreen.png" alt="Install Nextcloud Screen" id="Install Nextcloud Screen" >}}
+Accept the remaining defaults, and if setting up a cron job schedule, select **Enabled** under **Cron** to show the settings to allow you to schedule a cron job.
```

Comment:
"Accept the remaining defaults" is a little confusing, because it feels like that's it, I'm done setting up the app, but then we're really just eliding a few fields before talking about others that do need attention.

Reply:
changed this in the next commit